src/internal/deploy/assets/etcd.go

```diff
@@ -355,11 +358,7 @@ func EtcdVolumeClaim(size int, opts *AssetOpts) *v1.PersistentVolumeClaim {
 
 // EtcdNodePortService returns a NodePort etcd service. This will let non-etcd
 // pods talk to etcd
-func EtcdNodePortService(local bool, opts *AssetOpts) *v1.Service {
-	var clientNodePort int32
```

Comment:
We were setting these nodeports in kind of an obscure way - I changed this to only set the ports in dev mode, see below.